tests/test_live_validation.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * This file is part of RTRlib.
+ *
+ * INET group, Hamburg University of Applied Sciences,
+ * CST group, Freie Universitaet Berlin
+ * Website: http://rpki.realmv6.org/
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include "rtrlib/rtrlib.h"
+
+#define RPKI_CACHE_HOST "141.22.28.222"
```

Comment:
While coding I just remembered the IP, didn't know the corresponding URL anymore. Will fix that.